homeassistant/components/marstek/strings.json

```
@@ -0,0 +1,32 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "Device is already configured"
```

We have a common string for that:

```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```